README.md

```diff
@@ -187,46 +219,32 @@ Next, generate a Personal Access Token for bootstrapping the `core` packages, as
+With Habitat *0.88.0*, two new commands were introduced to assist in bootstrapping an on-prem Builder instance with a set of stable packages:
 
-Chef Habitat Builder on-prem has no pre-installed packages. To bootstrap a set of stable `core` origin packages (refer to the [core-plans repo](https://github.com/habitat-sh/core-plans)), you can do the following:
+1. *hab pkg download*
+1. *hab pkg bulkupload*
 
-1. Export your Personal Access Token as `HAB_AUTH_TOKEN` to your environment
+As you can see from the commands above, the package Bootstrap flow is comprised of two main phases: a download from the public [SaaS Builder](https://bldr.habitat.sh) followed by a bulkupload to your on-prem Builder instance(s). Historically, we bootstraped on-prem-builders by downloading all the packages in 'core' for all targets. That amounted to ~15GB and was both too much and too little, in that many of the packages weren't needed, and for many patterns (effortless) other origins were needed.
 
-    ```bash
-    export HAB_AUTH_TOKEN=<your token>
-    ```
+The [new bootstrap process flow](https://forums.habitat.sh/t/populating-chef-habitat-builder-on-prem/1228) allows you to easily customize your Bootstrap package set or use pre-populated [Starter Kit](https://github.com/habitat-sh/on-prem-builder/tree/master/quickstart_lists) files, which should amount to substantially less time and resource waste.
 
-1. Run the population script, passing the root URL of your new Chef Habitat Builder on-prem as the last argument (Replace `http` with `https` in the URL if SSL is enabled)
+The following section illustrates the steps required to bootstrap the on-prem Builder with the [Effortless Linux](https://github.com/habitat-sh/on-prem-builder/blob/master/quickstart_lists/effortless_x86_64-linux_stable) starter kit. Simply repeat the following download/bulkupload flow for the starter kits you think you will need to have in your on-prem Builder, or even create your own custom starter kit file:
 
+1. Phase 1: download
     ```bash
-    sudo -E ./scripts/on-prem-archive.sh populate-depot http://${BUILDER_HOSTNAME_OR_IP}`
+    export HAB_AUTH_TOKEN=<your _public_ Builder instance token>
+    cd on-prem-builder
+    hab pkg download --target x86_64-linux --channel stable --file quickstart_lists/effortless_x86_64-linux_stable --download-directory builder_bootstrap
+    ```
+1. Phase 2: bulkupload
+*Important*: Inspect the contents of the `builder_bootstrapi/artifacts` directory created from the download command above. For each of the origins (`core`, `effortless`, etc),  create a matching origin name if one doesn't exist already in the on-prem Builder UI before starting the bulkupload.
+    ```bash
+    export HAB_AUTH_TOKEN=<your _on-prem_ Builder instance token>
+    hab pkg bulkupload --url https://your-builder.tld --channel stable builder_bootstrap/
     ```
```

Comment:
Instead of completely removing the Synchronization section, we should still have the section topic, but change the prescriptive guidance to use the download/bulkupload commands

Reply:
Handled in https://github.com/habitat-sh/on-prem-builder/pull/212/commits/634b062e33a6a3d4aede2d6788fc9bf0f4160152

I re-added the section with a deprecation notice:

## Bootstrap `core` packages (**Deprecated**)